backend/models/class.model.ts

```diff
@@ -0,0 +1,35 @@
+import mongoose, { Schema, Document, model } from "mongoose";
+
+export interface Class extends Document {
+  class_name: string;
+  school_year: number;
+  grade_level: string;
+  teacher: string;
+  test_sessions: string[];
+}
+
+const ClassSchema: Schema = new Schema({
+  class_name: {
+    type: String,
+    required: true,
+  },
+  school_year: {
+    type: Number,
+    required: true,
+  },
+  grade_level: {
+    type: String,
```

Comment:
can you change the `gradeLevel` field type to be an enum from "K" - "Grade 8"? you can look at `user.model.ts` as a reference